server/shared/common.js

```diff
@@ -32,6 +32,7 @@ export const dtnloss = 8
 export const dtcloss = 9
 export const dtloh = 10 // to be used in svcnv track
 export const plotColor = '#ce768e'
+export const dtmetaboliteintensity = 11
```

Comment:
yes i agree with this design
later can introduce `dtdictionaryterms` and `dtsinglecellgeneexp` and support those in hiercluster